lms/static/js/spec/learner_dashboard/course_entitlement_view_spec.js

```diff
@@ -1,158 +1,158 @@
-                )).toBe(true);
-            });
-        });
-
-        describe('Available Sessions Select - Fulfilled Entitlement', function() {
-            beforeEach(function() {
-                setupView(true);
-                selectOptions = view.$('.session-select').find('option');
-            });
-
-            it('Select session dropdown should show available course runs, coming soon and leave options.', function() {
-                expect(selectOptions.length).toEqual(entitlementAvailableSessions.length + 2);
-            });
-
-            it('Select session dropdown should allow user to leave the current session.', function() {
-                var leaveSessionOption = selectOptions[selectOptions.length - 1];
-                expect(leaveSessionOption.text.includes('Leave the current session and decide later')).toBe(true);
-            });
-
-            it('Currently selected session should be specified in the dropdown options.', function() {
-                var selectedSessionIndex = _.findIndex(entitlementAvailableSessions, function(session) {
-                    return initialSessionId === session.session_id;
-                });
-                expect(selectOptions[selectedSessionIndex].text.includes('Currently Selected')).toBe(true);
-            });
-
-            it('Title element should correctly indicate the expected behavior.', function() {
-                expect(view.$('.action-header').text().includes(
-                    'Change to a different session or leave the current session.'
-                )).toBe(true);
-            });
-        });
-
-        describe('Select Session Action Button and popover behavior - Unfulfilled Entitlement', function() {
-            beforeEach(function() {
-                setupView(false);
-            });
-
-            it('Change session button should have the correct text.', function() {
-                expect(view.$('.enroll-btn-initial').text() === 'Select Session').toBe(true);
-            });
-
-            it('Select session button should show popover when clicked.', function() {
-                view.$('.enroll-btn-initial').click();
-                expect(view.$('.verification-modal').length > 0).toBe(true);
-            });
-        });
-
-        describe('Change Session Action Button and popover behavior - Fulfilled Entitlement', function() {
-            beforeEach(function() {
-                setupView(true);
-                selectOptions = view.$('.session-select').find('option');
-            });
-
-            it('Change session button should show correct text.', function() {
-                expect(view.$('.enroll-btn-initial').text().trim() === 'Change Session').toBe(true);
-            });
-
-            it('Switch session button should be disabled when on the currently enrolled session.', function() {
-                expect(view.$('.enroll-btn-initial')).toHaveClass('disabled');
-            });
-        });
-    });
-}
-);
+// define([
```

Comment:
Please add a comment explaining why this is commented out. It could be confusing for other contributors.